examples/colors.cpp

```diff
@@ -1,6 +1,6 @@
-#include <cpp-terminal/version.h>
-#include <cpp-terminal/base.hpp>
 #include <iostream>
+#include "cpp-terminal/base.hpp"
```

Comment:
Shouldn't this be using <> since it's included through `/usr/include` or using cmake?

Reply:
Yes, sorry for the commit mess, I though only small change was needed, but has I could not test the CI. I had this ugly commits... I'm not expert on git stufs. I need to dig on the git machinery to have a better understanding of the more advances features.  

For the <> and "" yes I think you are right it should be <> but the cases can be different when compiling the library and when using it.. It can stay like this for now maybe has we have a special issue related to this

Reply:
No worries about the commit messages. I'll just do a squash when merging ^^ I moved the discussion to #197.